biome.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.8.3/schema.json",
+  "organizeImports": {
+    "enabled": true
+  },
+  "linter": {
+    "enabled": true,
+    "rules": {
+      "recommended": true
+    }
+  },
+  "formatter": {
+    "enabled": true,
+    "indentStyle": "space",
+    "indentWidth": 2,
+    "lineEnding": "lf",
+    "lineWidth": 80
```

Comment:
what year is it, what should we do with all the 42inch screens? 😤

Reply:
I'm more than open to change it - I have created this config by running the [migrate](https://biomejs.dev/guides/migrate-eslint-prettier/) command from Biome, so I assume this width was the default in either Eslint or Biome.